src/main/java/Presenter/spacePresenter.java

```diff
@@ -6,14 +6,24 @@
 
 import java.util.ArrayList;
 
+/**
+ * presenter for the space use cases.
+ */
 public class spacePresenter implements songOutputBoundary, spacePlayedOutputBoundary {
```

Comment:
remember to override the methods in songOB and spacePlayedOB. (ie, change the button and the playbar). if they are the same then merge